rust/xaynet-analytics/src/database/analytics_event/data_model.rs

```diff
@@ -13,55 +16,56 @@ pub enum AnalyticsEventType {
 }
 
 #[derive(Debug, PartialEq, Eq, Clone)]
-pub struct AnalyticsEvent {
+pub struct AnalyticsEvent<'a> {
```

Comment:
Since this lifetime is pretty pervasive in our code would be better to have a descriptive name. `screenroute` or `sr`.

Reply:
ok will do